src/VTMap.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Map, TileLayer, Polygon } from "react-leaflet";
+import borderData from "./border";
+import Buttons from "./Buttons";
+import Modals from "./Modals";
+import L from "leaflet";
+import leafletPip from "@mapbox/leaflet-pip";
+import Navigation from "./Navigation";
+/* Game Start Coordinates*/
+class VTMap extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      center: [44.0886, -72.7317],
+      zoom: 7,
+      style: { height: "600px", width: "600px" },
+      zoomControl: false,
+      scrollWheelZoom: false,
+      attributionControl: false,
+      disabled: true,
+    };
+  }
+  /*Click Handler for Game Start*/
+  handleClick = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      disabled: false,
+      gameStarted: true,
+      zoom: 18,
+    });
+  };
+/* Quit Button Handler*/
+  handleQuitButton = (event) => {
+    event.preventDefault();
+
+    this.setState({
+      gameStarted: false,
+      zoom: 7,
+    });
+  };
+ /* Fetches Coordinates for Guess*/
+  getCurrentLoc = (event) => {
+    let zoomCoords = randomCoords();
+    this.handleClick(event);
+    let lat = zoomCoords[0];
+    let long = zoomCoords[1];
+    let request = `https://nominatim.openstreetmap.org/reverse?lat=${lat}&lon=${long}&format=geojson`;
+    this.setState({
+      center: zoomCoords,
+    });
```

Comment:
When setting up your movement buttons look at what you're doing here to move the map from being centered on VT to a random location. You're updating the state with a new set of `[lat, lng]` coords to move the map. I find .002 degrees is a good distance for moving the map, though that might need tweaking depending on the size of the map.